docs/_risks/ri-24_agent-action-authorization-bypass.md

```diff
@@ -0,0 +1,74 @@
+---
+sequence: 24
+title: Agent Action Authorization Bypass
+layout: risk
+doc-status: Draft
+type: SEC
+owasp-llm_references:
+  - llm06-2025  # LLM06:2025 Excessive Agency
+related_risks:
+  - ri-10  # Prompt Injection
+  - ri-18  # Model Overreach / Expanded Use
+  - ri-22  # Regulatory Compliance and Oversight
+---
+
+## Summary
+
+Agent systems may bypass intended authorization controls and perform actions beyond their designated scope, potentially executing unauthorized financial transactions, accessing restricted data, or violating business logic constraints. This occurs when agents exploit API vulnerabilities, escalate privileges through tool chains, or circumvent approval workflows designed to maintain segregation of duties and regulatory compliance.
+
+## Description
+
+Agentic AI systems in financial services operate with significantly more autonomy than traditional RAG-based implementations, capable of making decisions and executing actions through various APIs and tools. This autonomy introduces a critical security risk: **Agent Action Authorization Bypass**, where agents perform operations outside their intended authorization boundaries.
```

Comment:
I think the important thing to highlight is that they _operate with autonomy_, whereas most other systems lack any form of autonomy, typically following workflows guided by hard-coded rules.